Gordon360/Controllers/CliftonStrengthsController.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Gordon360.Models.CCT.Context;
+using Gordon360.Models.ViewModels;
+using Gordon360.Services;
+using Gordon360.Static.Names;
+using Gordon360.Authorization;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System;
+using Gordon360.Models.CCT;
+using Gordon360.Exceptions;
+
+namespace Gordon360.Controllers
+{
+    [Route("api/[controller]")]
+    public class CliftonStrengthsController : GordonControllerBase
+    {
+        private CCTContext _context;
+        private IAccountService _accountsService;
+
+        public CliftonStrengthsController(CCTContext context, IAccountService accountService)
+        {
+            _context = context;
+            _accountsService = accountService;
+        }
+
+        [HttpPost]
+        [Route("")]
+        [StateYourBusiness(operation = Operation.READ_PARTIAL, resource = Resource.MEMBERSHIP_BY_ACTIVITY)]
+        public ActionResult<IEnumerable<CliftonStrengthsUploadViewModel>> Post([FromBody] CliftonStrengthsUploadViewModel[] csArr)
```

Comment:
Could we use a more descriptive parameter name?
Maybe something like `strengthsToUpload`?

Reply:
Oh also, this return type is wrong.
```suggestion
        public ActionResult<IEnumerable<AddCliftonStrengthsViewModel>> Post([FromBody] CliftonStrengthsUploadViewModel[] csArr)
```